src/components/Contact/ContactForm.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
+import styles from "./ContactForm.module.css";
+import FormElement from "../utility/FormElement/FormElement";
+import Button from "../utility/Button/Button";
+
+const ContactForm = () => {
+  const [formData, setFormData] = useState({
+    name: { value: "", error: "", touched: false },
+    email: { value: "", error: "", touched: false },
+    companyName: { value: "", error: "", touched: false },
+    title: { value: "", error: "", touched: false },
+    message: { value: "", error: "", touched: false },
+  });
+
+  const handleChange = (inputField) => (e) => {
+    const { value } = e.target;
+    setFormData((prevState) => ({
+      ...prevState,
+      [inputField]: {
+        ...prevState[inputField],
+        value: value,
+        error: "",
+        touched: true,
+      },
+    }));
+  };
+
+  // check if email is valid
+  const isEmailValid = (email) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return emailRegex.test(email);
+  };
+
+  const handleBlur = (inputField) => () => {
```

Comment:
```suggestion
  const handleBlur = (inputField)=> {
```
I think you meant this

Reply:
no the second one is added to make it a higher order function while the outer function specify which input field  the event applies to the inner one handles the blur event